src/app/pages/ArticlePage/types.ts

```diff
@@ -8,3 +8,13 @@ export type TimeStampProps = {
   firstPublished: number;
   lastPublished: number;
 };
+
+export type getHeadlineComponentProps = {
+  pathname: string; 
+  isLite: boolean;
+  translations?: {
+    liteSite?: {
+      articleDataSavingLinkText?: string;
+    };
```

Comment:
Rather than duplicating the type definition, could we consider this?

Translations type from https://github.com/bbc/simorgh/blob/3ff04cd8478850a064582f134a7636f2e74ee6f1/src/app/models/types/translations.ts#L2

```suggestion
  translations? Translations
```